src/tactic/interactive_expr.lean

```diff
@@ -39,6 +39,8 @@ meta inductive sf : Type
 | tag_expr : expr.address → expr → sf → sf
 | compose : sf →  sf →  sf
 | of_string : string →  sf
+| highlight : format.color → sf → sf
```

Comment:
Is `highlight` actually used?  I was under the impression that this is just a vestigial part of the format code copied from Z3.

Reply:
No it's not used, I looked at the C++ code base, the constructor for it was never called. I added this mostly because it was easy to do, and I took it that it was an oversight in the C++ code base rather than a deliberate removal. 

Do you want me to undo this change?

Reply:
I don't think it is used anywhere currently, but it seems like a nice thing to have!